src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -447,6 +447,13 @@
     "NAME": "Scorching Ray",
     "DESCRIPTION": "Remember stsjorbsmod:Lust. NL Apply !stsjorbsmod:MetaMagic! stsjorbsmod:Burning !M! times. This stsjorbsmod:Burning is affected by Strength."
   },
+  "stsjorbsmod:Seance": {
+    "NAME": "Seance",
+    "DESCRIPTION": "Put a card from your exhaust pile into your hand. It gains Ethereal. NL Exhaust",
```

Comment:
We have an `stsjorbsmod:Exhume` keyword that could shorten this description a bit